boards/BMS/Src/Io/Io_Latches.c

```diff
@@ -0,0 +1,28 @@
+#include <stm32f3xx.h>
+#include "main.h"
+#include "Io_Latches.h"
+
+void Io_Latches_EnableOnBmsOk(void)
```

Comment:
`Io_Led` has been superseded by `Io_Latches`. Some context: The signal `BMS_OK`, `IMD_OK`, and `BSPD_OK` are inputs to a D-latch. It is also true that these signals are connected to LEDs for debugging purposes. However, it is more useful for BMS to treat these signals as D-Latch inputs rather than LED inputs.